### DISCUSSION TITLE
Support for QA: Active Mode to index TXT Files.

### DISCUSSION THREAD

**Discussion by ghost** [logancyang/obsidian-copilot#143] [Ideas]
I enabled the Txt as Md community Plugin, which allows me to edit and work with txt files in Obsidian. It would be nice if the Copilot were able to view the txt file. As of now, only .md is supported.

**Comment by logancyang** [comment]
Right I'm thinking about enabling other files. @Sokole1 has a PR for PDF support. If we could just refer to any file by its title in the chat it would be nice.

**Comment by Sokole1** [comment]
Hi @ionaprotocol, I downloaded the txt as md plugin and with a small change, I can support .txt as well in my PR (#131) which will work with all the various features: https://github.com/Sokole1/obsidian-copilot/commit/24a9e11b509c4296a1e07aea8675fe7e08264fc8
